mergeSort.js

```diff
@@ -1,3 +1,4 @@
+// merge sort algorithm
 function mergeSort(arr) {
```

Comment:
Insert newline after the closing brace of the if statement.
In the while loop, this would the one time you want a if and else if being used, rather than two if's side by side, since they could both fire within the same while loop iteration.
In the while loop, instead of splicing, see if there's an array method that handle the specific case of removing something from the beginning of the array.